.ci/static.sh

```diff
@@ -18,11 +18,15 @@ if [[ "$COMMAND" == "install" ]]; then
     conda install jupyter
     pip install codespell flake8 pylint
 elif [[ "$COMMAND" == "run" ]]; then
+    # Convert notebooks to Python scripts
     jupyter-nbconvert \
         --log-level WARN \
         --to python \
         --TemplateExporter.exclude_input_prompt=True \
         -- **/*.ipynb
+    # Remove style issues introduced in the conversion:
+    #   s/# $/#/g replaces lines with just '# ' with '#'
```

Comment:
Yeah for some reason when nbconvert converts markdown cells, it makes everything a comment, and when it makes empty lines into comments it does it by having the line contain `# ` rather than just `#`, and `# ` is a style issue because of trailing whitespace.